### PR TITLE
Make toast copy more consistent

### LINKED ISSUE #2499: Consolidate copy and approach to toast copy
We're a little inconsistent with toast copy and CTAs at the moment. We should do a pass on consolidating them all. E.g. settling on capitalisation, use of quotes, and when to link to the resource that has been created / modified.

### PR DESCRIPTION
**Pull request by charliepark**
This makes the toasts across the app more consistent, and adds highlighting to the resource name (when available) for each of the toasts.
Screenshot 2024-10-17 at 6 19 58 AM
Screenshot 2024-10-17 at 6 30 41 AM

Closes #2499

### CODE REVIEW

File: app/forms/disk-create.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ export function CreateDiskSideModalForm({
   const createDisk = useApiMutation('diskCreate', {
     onSuccess(data) {
       queryClient.invalidateQueries('diskList')
-      addToast({ content: 'Your disk has been created' })
+      addToast({ content: `${data.name} created` })
```
Comment (david-crespo) [inline app/forms/disk-create.tsx:79]: These names might benefit from something like the `HL` treatment we currently use in some modals.

image


https://github.com/oxidecomputer/console/blob/a76ba5f46f2509d21c56d397e1f1cf62bd8dbbbb/app/pages/project/instances/instance/tabs/NetworkingTab.tsx#L349-L362

**Comment by david-crespo** [comment]
Love the direction, thanks for doing this tedious stuff.

**Comment by charliepark** [comment]
The latest, with highlighting, and removing the `Success` text:
Screenshot 2024-10-15 at 5 42 38 PM

**Comment by david-crespo** [comment]
Prettier is killing me. This sent me on an odyssey to see if `biome` or `dprint` could give me

```ts
addToast({ content: <>IP <HLs>{ephemeralIp.ip}</HLs> attached</> })
```

but neither one can. Reconsidering my choice of career.

**Comment by david-crespo** [comment]
The highlighting might make the secondary color text look too dim.



Here's what it looks like with both in `text-accent`. Also kinda meh because the highlight doesn't stand out. @benjaminleonard help

**Comment by benjaminleonard** [comment]
I think the semi is exaggerating that there. Try removing that and keeping primary and secondary as the main mode of contrast.

**Comment by charliepark** [comment]
Thoughts?
Screenshot 2024-10-17 at 5 47 11 AM
Screenshot 2024-10-17 at 5 47 58 AM

**Comment by charliepark** [comment]
Screenshot 2024-10-17 at 6 17 33 AM
Screenshot 2024-10-17 at 6 18 02 AM
Screenshot 2024-10-17 at 6 18 29 AM
Screenshot 2024-10-17 at 6 18 43 AM
Screenshot 2024-10-17 at 6 19 58 AM

File: app/components/HL.tsx [hunk 2]
```diff
@@ -7,4 +7,11 @@
...
 
-export const HL = classed.span`text-sans-semi-md text-default`
+// note parent with secondary text color must have 'group' on it for
+// this to work. see Toast for an example
+export const HL = classed.span`
+  text-sans-md text-default 
+  group-[.text-accent-secondary]:text-accent
+  group-[.text-error-secondary]:text-error
+  group-[.text-info-secondary]:text-info
+`
```
Comment (david-crespo) [inline app/components/HL.tsx:17]: Mad scientist stuff. This is the CSS selector it generates.

```css
.group.text-accent-secondary .group-\[\.text-accent-secondary\]\:text-accent {
  // ...
}
```

**Comment by david-crespo** [comment]
oh wild. I just got the error locally, so maybe it's not the runner!

image

**Comment by david-crespo** [comment]
ok lmao. in a3d736b04977c43a6d673e3ae4977e6a606cff71, I did a giant find-replace of `HLs` to `HL` and this hit a few hashes in the lockfile. Good prank, universe.

image

**Review by david-crespo — APPROVED** [review]
Thanks for doing all the fiddly work @charliepark. The toasts have been bothering me forever!

**Pull request merged**